packages/iov-bns/src/decodemsg.ts

```diff
@@ -257,6 +348,20 @@ export function decodeMsg(base: UnsignedTransaction, tx: BnsdTxMsg): UnsignedTra
   if (tx.aswapReleaseMsg) return decodeSwapClaimTx(base, tx.aswapReleaseMsg);
   if (tx.aswapReturnMsg) return decodeSwapAbortTransaction(base, tx.aswapReturnMsg);
 
+  // Term Deposits
+  if (tx.termdepositUpdateConfigurationMsg) {
+    return decodeUpdateTermDepositConfigurationTx(base, tx.termdepositUpdateConfigurationMsg);
+  }
+  if (tx.termdepositCreateDepositContractMsg) {
+    return decodeCreateTermDepositContractTx(base, tx.termdepositCreateDepositContractMsg);
+  }
+  if (tx.termdepositDepositMsg) {
+    return decodeTermDepositTx(base, tx.termdepositDepositMsg);
+  }
+  if (tx.termdepositReleaseDepositMsg) {
```

Comment:
`termdepositReleaseDepositMsg` should be `termdepositReleaseMsg` for consistency.

Reply:
I can not change it, this naming is come from protobuf interface description